frontend/src/components/ExportPreview/ExportPreview.tsx

```diff
@@ -1,16 +1,16 @@
 import React, { useEffect, useState, useRef } from 'react';
-import ScheduledTable from '../UserTables/ScheduledTable';
 import moment from 'moment';
+import { CSVLink } from 'react-csv';
+import ScheduledTable from '../UserTables/ScheduledTable';
 import { Driver } from '../../types/index';
 import styles from './exportPreview.module.css';
 import { useReq } from '../../context/req';
 import ExportButton from '../ExportButton/ExportButton';
 import { useDate } from '../../context/date';
-import { CSVLink } from "react-csv";
 
 const ExportPreview = () => {
   const [drivers, setDrivers] = useState<Driver[]>([]);
-  const [downloadData, setDownloadData] = useState<string>("");
+  const [downloadData, setDownloadData] = useState<string>('');
```

Comment:
Is there any reason to use single quotes vs double?

Reply:
single quotes is whats part of our eslint rule, it just auto corrected on save